sustaingym/envs/cogen/env.py

```diff
@@ -65,25 +65,26 @@ class CogenEnv(gym.Env):
         Natural gas price ($/MMBtu)   0                   7
 
     Args:
-        renewables_magnitude: TODO
-        ramp_penalty: TODO
-        supply_imbalance_penalty: TODO
-        constraint_violation_penalty: TODO
-        forecast_horizon: TODO
-        forecast_noise_std: TODO
+        renewables_magnitude (float): wind generation capacity
```

Comment:
For these args, you don't need to specify the type, because the typing annotations in the `__init__` function already tell you the expected type